packaging/debian-unstable/tests/control

```diff
@@ -0,0 +1,9 @@
+Tests: integrationtests
+Restrictions: allow-stderr, isolation-container, rw-build-tree, needs-root, breaks-testbed
+Depends: @builddeps@,
+         bzr,
```

Comment:
I think bzr is not really needed anymore. Last time I looked at arch packaging I dropped it.